### PR TITLE
EvaluatedModel: Add `summary` and `description` of vulnerabitlies

### PR DESCRIPTION
**Pull request by fviernau**
This prepares for showing these properties in the WebApp / UI.
See individual commits.

Context: #5593

### CODE REVIEW

File: reporter/src/funTest/kotlin/reporters/evaluatedmodel/EvaluatedModelReporterFunTest.kt [hunk 1]
```diff
@@ -81,3 +78,5 @@ private fun TestConfiguration.generateReport(ortResult: OrtResult, options: Map<
 
     return EvaluatedModelReporter().generateReport(input, outputDir, options).single().readText()
 }
+
+private fun getAssetAsString(path: String): String = File("src/funTest/assets").resolve(path).readText()
```
Comment (sschuberth) [inline reporter/src/funTest/kotlin/reporters/evaluatedmodel/EvaluatedModelReporterFunTest.kt:79]: I guess I'd rather move the existing `getAssetAsString()` from `OsvServiceFunTest` to the `test-utils` module and use it from there; that should work as [all our library project have `test-utils` as a (fun)test dependency](https://github.com/oss-review-toolkit/ort/blob/6bff25b967a30d56c07c5328a6b2de3741dacccc/build.gradle.kts#L191-L193).

File: reporter/src/funTest/assets/evaluated-model-reporter-test-expected-output.json [hunk 2]
```diff
@@ -586,7 +586,7 @@
       "path" : ""
     },
     "paths" : [ 3, 4 ],
-    "levels" : [ 1 ],
+    "levels" : [ 1, 2 ],
```
Comment (sschuberth) [inline reporter/src/funTest/assets/evaluated-model-reporter-test-expected-output.json:589]: Commit message nits:
- "by instead" -> "and instead"
- "inline" -> "in line"

File: reporter/src/funTest/kotlin/reporters/evaluatedmodel/EvaluatedModelReporterFunTest.kt (hunk 1 above)
Comment (fviernau) [reply reporter/src/funTest/kotlin/reporters/evaluatedmodel/EvaluatedModelReporterFunTest.kt:79]: Oh, I wasn't aware of the existence of this project. Thanks for pointing it out!

File: utils/test/src/main/kotlin/Utils.kt [hunk 3]
```diff
@@ -115,3 +115,8 @@ fun <T, U> transformingCollectionMatcher(
 fun <T, U> transformingCollectionEmptyMatcher(
     transform: (T) -> Collection<U>
 ): Matcher<T?> = neverNullMatcher { value -> beEmpty<U>().test(transform(value)) }
+
+/**
+ * Return the asset file located under [path] relative to the 'assets' directory as text.
+ */
+fun getAssetAsString(path: String): String = File("src/funTest/assets").resolve(path).readText()
```
Comment (sschuberth) [inline utils/test/src/main/kotlin/Utils.kt:122]: Please maintain the alphabetical ordering of functions in this file.

**Review by sschuberth — DISMISSED** [review]

**Review by sschuberth — APPROVED** [review]

**Pull request merged**